src/components/AddItemForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+import '../CSS/AddItemForm.css';
+
+// const initialState = {
+
+// }
+class AddItemForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      next_purchase: 14,
+      last_purchase: null,
+      useToken: 'shpongle',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSchedule = this.handleSchedule.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleSchedule(event) {
+    this.setState({ next_purchase: event.target.value });
+  }
+
+  handleSubmit(event) {
+    const { name, next_purchase, last_purchase } = this.state;
```

Comment:
Sorry I didn't mention this in our Slack thread, but I believe you want to store the user's token along with the other data points, correct?